CHANGELOG.md

```diff
@@ -352,8 +352,8 @@ gmond.conf
 
  * [972e5ad](../../commit/972e5ad) - __(Jonathan Beckman)__ Removed lies from gmetad config for el6
 
-By default EL6 ganglia runs as nobody and not ganglia.
-This fixes the issue by changing the user that ganglia is
+By default EL6 ganglia runs as nobody and not ganglia.
+This fixes the issue by changing the user that ganglia is
```

Comment:
There's a CRLF issue that git kept flagging.  I just ran that file through dos2unix.